src/workerd/api/sql.c++

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "sql.h"
+#include <stdio.h>
```

Comment:
Nit: is this file actually using `stdio`? I missed it if it is.

Reply:
FWIW if you included this for some printf debugging, check out `KJ_DBG` as a much better alternative.